src/NServiceBus.Core/Routing/AutomaticSubscriptions/AutoSubscribe.cs

```diff
@@ -43,7 +43,7 @@ protected internal override void Setup(FeatureConfigurationContext context)
 
                     var messageTypesHandled = GetMessageTypesHandledByThisEndpoint(handlerRegistry, conventions, settings);
 
-                    return new ApplySubscriptions(messageTypesHandled);
+                    return new ApplySubscriptions(messageTypesHandled, type => Task.FromResult(true));
```

Comment:
@danielmarbach can you raise an issue in the NSB repo for that? I agree it is wrong.